src/ota.h

```diff
@@ -13,6 +14,11 @@
 #define MG_OTA MG_OTA_NONE
 #endif
 
+#if defined(__GNUC__) && !defined(__APPLE__)
+#undef MG_IRAM
+#define MG_IRAM __attribute__((section(".iram")))
+#endif
+
```

Comment:
I see all ota functions (most ?) are prefixed with this.
Do all of them need to run in RAM ? Maybe not the case for dual bank devices, or are those not called/used with dual bank devices ?

Reply:
Only if the device is single bank do they need to run in RAM. Perhaps we could also add a condition on line 17 to check if the device is single bank?